packages/opal-server/opal_server/config.py

```diff
@@ -250,65 +316,102 @@ class OpalServerConfig(Confi):
+    )
+    BUNDLE_IGNORE = confi.list(
+        "BUNDLE_IGNORE",
+        [],
+        description="List of patterns to ignore when creating policy bundles. Similar to .gitignore, this allows excluding certain files or directories from policy bundles."
+    )
 
-    NO_RPC_LOGS = confi.bool("NO_RPC_LOGS", True)
+    NO_RPC_LOGS = confi.bool(
+        "NO_RPC_LOGS",
+        True,
+        description="Disable RPC logs to reduce log verbosity. Set to False if you need detailed logging of RPC calls for debugging."
+    )
 
     # client-api server
     SERVER_WORKER_COUNT = confi.int(
         "SERVER_WORKER_COUNT",
         None,
-        description="(if run via CLI) Worker count for the server [Default calculated to CPU-cores]",
+        description="Number of worker processes for the server. If not set, it defaults to the number of CPU cores. More workers can handle more concurrent requests."
     )
 
     SERVER_HOST = confi.str(
         "SERVER_HOST",
         "127.0.0.1",
-        description="(if run via CLI)  Address for the server to bind",
+        description="Address for the server to bind to. Use '0.0.0.0' to bind to all available network interfaces."
     )
 
     SERVER_PORT = confi.str(
         "SERVER_PORT",
         None,
-        # Users have experienced errors when kubernetes sets the env-var OPAL_SERVER_PORT="tcp://..." (which fails to parse as a port integer).
-        description="Deprecated, use SERVER_BIND_PORT instead",
+        description="Deprecated. Use SERVER_BIND_PORT instead. Kept for backward compatibility."
     )
 
     SERVER_BIND_PORT = confi.int(
         "SERVER_BIND_PORT",
         7002,
-        description="(if run via CLI)  Port for the server to bind",
+        description="Port for the server to bind to. This is the port on which OPAL server will listen for incoming connections."
     )
 
     # optional APM tracing with datadog
     ENABLE_DATADOG_APM = confi.bool(
         "ENABLE_DATADOG_APM",
         False,
-        description="Set if OPAL server should enable tracing with datadog APM",
+        description="Enable tracing with Datadog APM (Application Performance Monitoring). Useful for monitoring OPAL's performance in production environments."
     )
 
-    SCOPES = confi.bool("SCOPES", default=False)
+    # Scopes feature
+    SCOPES = confi.bool(
+        "SCOPES",
+        default=False,
+        description="Enable the scopes feature. Scopes allow for more granular control over policy and data distribution."
+    )
 
     SCOPES_REPO_CLONES_SHARDS = confi.int(
         "SCOPES_REPO_CLONES_SHARDS",
         1,
-        description="The max number of local clones to use for the same repo (reused across scopes)",
+        description="The maximum number of local clones to use for the same repo (reused across scopes). This helps in managing multiple scopes efficiently."
```

Comment:
@onyedikachi-david this one needs more explanation. what are those shards and their relationship to the scopes